### PR TITLE
1568 Make Zoe use AVA

### PR DESCRIPTION
**Pull request by dtribble**
ref #1568

Kate: All tests pass. Most recent changes to tests were caused by @Chris-Hibbert's changes to the autoswap tests. Chris could you take a look at test-autoswap.js specifically?

### CODE REVIEW

**Comment by warner** [comment]
Ok @katelynsills , I've had three successful test runs with that ESM_DISABLE_CACHE patch, so I'm hoping the problem is fixed. Handing this back to you for rebasing and landing.

**Comment by katelynsills** [comment]
@warner, wonderful, thanks so much!

**Review by Chris-Hibbert — APPROVED** [review]
Thanks for taking care of this!  The changes to autoswap look good to me.

File: packages/zoe/test/swingsetTests/zoe/test-zoe.js [hunk 1]
```diff
@@ -298,12 +268,12 @@ const expectedBadTimerLog = [
   'aliceMoolaPurse: balance {"brand":{},"value":3}',
   'aliceSimoleanPurse: balance {"brand":{},"value":0}',
 ];
-test('zoe - bad timer', async t => {
+test.skip('zoe - bad timer', async t => {
```
Comment (Chris-Hibbert) [inline packages/zoe/test/swingsetTests/zoe/test-zoe.js:273]: There should be an issue for this, and there should be a TODO here referring to the issue.

**Review by Chris-Hibbert — CHANGES_REQUESTED** [review]
skipped tests should have matching TODOs and issues.

File: packages/zoe/test/swingsetTests/zoe/test-zoe.js (hunk 1 above)
Comment (katelynsills) [reply packages/zoe/test/swingsetTests/zoe/test-zoe.js:273]: There is an issue. https://github.com/Agoric/agoric-sdk/issues/1625 I will add TODOs

**Review by Chris-Hibbert — APPROVED** [review]
Thanks!  good to hear the issue was created earlier.

**Pull request merged**